server/routes/shared/referralOverviewPagePresenter.ts

```diff
@@ -25,4 +25,6 @@ export default class ReferralOverviewPagePresenter {
       },
     ],
   }
+
+  readonly dashboardURL = `/${this.subNavUrlPrefix}/referrals/dashboard`
```

Comment:
would be good to test somewhere that these urls get constructed properly for both sp and pp versions of the page

Reply:
👍 done